courses/free-courses-id.md

```diff
@@ -109,6 +110,7 @@
 
 * [Belajar Go-Lang untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH_t5_dtCQZgWJqWF45WRgZw) - Programmer Zaman Now
 * [Belajar Golang Basic Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXDztO-BFc9R5afhP26Dhsgm) - Sekolah Koding
+* [Golang Fundamental](https://buildwithangga.com/kelas/golang-fundamental?thumbnail=nk4neM2UyG.54&main_leads=browse) - BuildWithAngga
```

Comment:
Creator: Agung Setiawan
Right link: https://buildwithangga.com/kelas/golang-fundamental

Use this snippet as fix:
```markdown
* [Golang Fundamental](https://buildwithangga.com/kelas/golang-fundamental) - Agung Setiawan *(BuildWithAngga account and email address required)*
```